README.md

```diff
@@ -7,36 +7,32 @@
 
 Standard, Specification, Schema
 
-### Gitter
+### Community Chat
 
-Everyone working on the early stages of the project should join our gitter channel [gitter.im/jsonresume/public](https://gitter.im/jsonresume/public).
+* [Gitter](https://gitter.im/jsonresume/public)
 
-### Getting started
+### Installation
 
+Resume-schema is intended to be installed as a dependency of your project:
+
+```sh
+npm add resume-schema
 ```
-npm install --save resume-schema
-```
 
-To use
```

Comment:
Since validation is only included in this repo for internal use (ie testing) it doesn't need a mention in the `README.md`

Reply:
Happy to remove this, but since we are moving validation from the package's exports, it seemed appropriate to add a section that describes how to use an example 3rd party validator. If you can ack this position and still want to remove, let me know and I'll do it.

Reply:
Great idea. With the exception of the CLI which has validation built-in we don't need to ship validation as an API.

But examples showing how to validate using common tools is :+1:

Also :+1: on an import example.

Mentioning how to import directly from the web would be good as well. I enabled GH Pages for the repo it should be directly linkable from there.

Reply:
> Mentioning how to import directly from the web would be good as well. I enabled GH Pages for the repo it should be directly linkable from there.

I'll leave that one to you. It's against my religion to encourage Github as a universal CDN for scripts.

Reply:
> we don't need to ship validation as an API.

That's the only motivation behind this PR. The rest of the changes accommodate that intention.

Reply:
So are we good on this one? Okay to mark as resolved?